### PR TITLE
Test first blog post

### PR DESCRIPTION
**Pull request by muralibasani**
Test blog post

### CODE REVIEW

**Comment by muralibasani** [comment]
@blomqma can you suggest what am I doing wrong here ? Build fails.
Basically I would like to have blog posts on the site.

**Comment by muralibasani** [comment]
> > @blomqma need your help. Can you suggest what am I doing wrong here ? Build fails. Basically I would like to have blog posts on the site.
> > It needs an extension 'ablog'
> 
> Hey, you seem to have a typo in the `ablog` module name, the name of the module is `blog`. I pushed one example commit where the build should work with the blog module and the blog list and example blog should render.
> 
> EDIT: nvm, I guess the module name didn't matter but commenting out the `ablog` extension seemed to work. Now the build is green with the blog posts.

thanks.

**Comment by muralibasani** [comment]
@blomqma one more help. I tried to keep the same theme furo, but running into some dependency issues. Can you pls take a look on that one too

**Comment by muralibasani** [comment]
@blomqma https://github.com/damian-krawczyk/sphinx-ablog-furo/blob/main/docs/conf.py

**Comment by muralibasani** [comment]
@blomqma updated to a different theme : pydata_sphinx_theme didn't need any stying changes pls take a look
Sidebar and content looks great

File: docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+.. post:: May 15, 2023
+   :tags: kafka,governance,topics,acls,klaw
+   :category: kafka
+   :author: Murali Basani
+
+Background
+----------
+
+    Kafka is widely adopted around the world due to its scalability and unique features that set it apart from traditional
+messaging. Many organizations are now using Kafka as part of their technology stack. However, one common challenge that
```
Comment (aindriu-aiven) [inline docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst:14]: The First line is printing out strangely like a quote.

File: docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+Some important considerations when managing Kafka topics include:
+
+- Who is allowed to produce and consume messages on a given topic and who owns it?
+- How do you backup and restore Kafka's configuration in case of failure?
+- Is security properly enforced through subscriptions and access controls?
+- How to smoothly promote topics with tested configurations from lower environments to production?
+- How to increase visibility and awareness of topics within the organization?
+
+
+    While it is manageable to have a couple of hundred kafka topics, but as they keep growing together with applications
```
Comment (aindriu-aiven) [inline docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst:27]: This one also comes out like a quote for some reason.

**Review by aindriu-aiven — COMMENTED** [review]
Each paragraph starts with a strange quoting that messes with the flow of the article.

Content looks nice though.

**Comment by blomqma** [comment]
The blog seems to work nicely with the new theme. Here are some bugs I noticed:
* The new theme adds the light/dark mode selector (it was hidden previously). When dark mode is enabled, it requires two clicks to change to light theme.
* The sidebar seems to have an unnecessary scroll bar.
* The GitHub icon button on the top-right corner seems to be smaller than the other buttons
* The home/landing page seems to be significantly narrower with the new theme, making e.g. the screenshot harder to see due to the smaller size.
* The landing page contains two footers on top of each other.

File: docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+- Who is allowed to produce and consume messages on a given topic and who owns it?
+- How do you backup and restore Kafka's configuration in case of failure?
+- Is security properly enforced through subscriptions and access controls?
+- How to smoothly promote topics with tested configurations from lower environments to production?
+- How to increase visibility and awareness of topics within the organization?
+
+
+While it is manageable to have a couple of hundred kafka topics, but as they keep growing together with applications
+and teams, it would certainly become hard to manage. There could be a need to isolate a bunch of topics from other teams.
+
```
Comment (aindriu-aiven) [inline docs/blog/2023/05/making-kafka-governance-breeze-for-developers.rst:36]: While a small number of topics may be manageable, as the use of Kafka grows along side additional applications and teams, it becomes harder to manage effectively and efficiently. Along with additional considerations of isolating sensitive topics that may contain GDPR, HIPPA or other sensitive data to only the applications that should have access to them.

**Comment by muralibasani** [comment]
> The blog seems to work nicely with the new theme. Here are some bugs I noticed:
> 
> * The new theme adds the light/dark mode selector (it was hidden previously). When dark mode is enabled, it requires two clicks to change to light theme.
> * The sidebar seems to have an unnecessary scroll bar.
> * The GitHub icon button on the top-right corner seems to be smaller than the other buttons
> * The home/landing page seems to be significantly narrower with the new theme, making e.g. the screenshot harder to see due to the smaller size.
> * The landing page contains two footers on top of each other.

fixed icon, footer, light/dark mode

scroll on sidebar seems to be  a problwm from theme
landing page width will be fixed soon. something on css

**Review by aindriu-aiven — APPROVED** [review]

**Pull request merged**